script/Arcueid/Day 9/09_00_ARC09_2_2.txt

```diff
@@ -18,8 +18,9 @@ Though, I'm not keen on letting her get too close a look at my body.
 [sha:549721d06378680354f23ed355f59151dcb3fefc]{
 -- Page 73, Offset 21495.
 -- 昨夜のノエル先生とのいざこざであちこち擦り剥いているし、万が一、内出血でも起こしていたら大事だ。
+// Blood blister perhaps more accurate, seek better term
 I have a number of scrapes and bruises from my scuffle with Noel-sensei last night,
- and if by some chance I did develop a little internal bleeding, I doubt I'll hear the end of it.
+ and if by some chance I did develop a little bruising, I doubt I'll hear the end of it.
```

Comment:
Subdermal hematoma is I think more what's going on here, basically just a real bad bruise. Internal bleeding reads significantly more severe. Maybe we should doublecheck intent with Petrikow.